### PR TITLE
Bug 1989724: Add Profiles to evict localStorage pods/ignore PVC pods

### PR DESCRIPTION
**Pull request by damemi**
Adds profiles which will enable these options in the descheduler policy
ref: https://issues.redhat.com/browse/WRKLDS-304

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/cc @ingvagabund @kasturinarra

File: manifests/4.9/kube-descheduler-operator.crd.yaml [hunk 1]
```diff
@@ -92,6 +92,10 @@ spec:
                   - TopologyAndDuplicates
                   - LifecycleAndUtilization
                   - DevPreviewLongLifecycle
+                  - EvictPodsWithLocalStorage
+                  - DoNotEvictPodsWithPVC
+                  type: string
```
Comment (ingvagabund) [inline manifests/4.9/kube-descheduler-operator.crd.yaml:97]: duplicated line

File: manifests/4.9/kube-descheduler-operator.crd.yaml [hunk 2]
```diff
@@ -92,6 +92,10 @@ spec:
...
+                type: array
```
Comment (ingvagabund) [inline manifests/4.9/kube-descheduler-operator.crd.yaml:98]: duplicated line

File: bindata/assets/profiles/EvictPodsWithLocalStorage.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+apiVersion: "descheduler/v1alpha1"
+kind: "DeschedulerPolicy"
+evictLocalStoragePods: true
```
Comment (ingvagabund) [inline bindata/assets/profiles/EvictPodsWithLocalStorage.yaml:3]: From https://github.com/openshift/cluster-kube-descheduler-operator/blob/c9a04369033fb1b4476d6577b60511f6cf25a866/pkg/operator/target_config_reconciler.go#L238:
```
// Merge will fill any empty for value type attributes on the dst struct using corresponding
// src attributes if they themselves are not empty. dst and src must be valid same-type structs
// and dst must be a pointer to struct.
// It won't merge unexported (private) fields and will do recursively any exported field.
func Merge(dst, src interface{}, opts ...func(*Config)) error {
	return merge(dst, src, opts...)
}
```

empty fields (i.e. ignorePvcPods and evictLocalStoragePods set to false) will get set to true if set through the profiles. Looks good.

**Review by ingvagabund — APPROVED** [review]

**Review by ingvagabund — CHANGES_REQUESTED** [review]
Removing the duplicated lines is the only nit

**Pull request merged**